sonar-ws/src/main/java/org/sonarqube/ws/client/ce/CeService.java

```diff
@@ -58,4 +62,13 @@ public TaskTypesWsResponse taskTypes() {
     return call(new GetRequest(path("task_types")), TaskTypesWsResponse.parser());
   }
 
+  /**
+   * Gets details of a Compute Engine task.
+   *
+   * @throws org.sonarqube.ws.client.HttpException if HTTP status code is not 2xx.
+   * @since 5.5
+   */
+  public WsCe.TaskResponse task(String id) {
```

Comment:
the pattern used is to have a TaskWsRequest POJO. It's debatable when you have only one field, but I would prefer to have the same pattern everywhere